plugins/inputs/phpfpm/phpfpm.go

```diff
@@ -96,8 +110,18 @@ func (g *phpfpm) Gather(acc telegraf.Accumulator) error {
 // Request status page to get stat raw data and import it
 func (g *phpfpm) gatherServer(addr string, acc telegraf.Accumulator) error {
 	if g.client == nil {
-		client := &http.Client{}
-		g.client = client
+		tlsCfg, err := g.ClientConfig.TLSConfig()
+		if err != nil {
+			return err
+		}
+		tr := &http.Transport{
+			ResponseHeaderTimeout: g.ResponseTimeout.Duration,
```

Comment:
This line is covered by the Client.Timeout so it can be removed.

Reply:
I suppose they do, though they serve different purposes. We should also clean it up from httpjson input when we check that next.

Reply:
The httpjson plugin is deprecated so I wouldn't worry about it.  This timeout is great if you are streaming the response body but otherwise its just simpler to use client timeout vs contemplating all the possible timeouts.  Just my 2 cents.